src/widgets/Header/ui/index.tsx

```diff
@@ -27,6 +31,8 @@ const Header = (): JSX.Element | null => {
 
   const onLogoClick = () => {
     if (user?.id) {
+      clearSelectedApplet()
+      clearActivity()
```

Comment:
It is not clear why we need to perform the clean up before navigating to Applets page